test/services/database-analyzer/database-analyzer-mongo.test.js

```diff
@@ -89,5 +109,125 @@ describe('services > database analyser > MongoDB', () => {
       expect(model).toStrictEqual(expectedManyObjectIDFieldsModel);
       await mongoHelper.close();
     });
+
+    it('should generate the model with a nested object', async () => {
+      expect.assertions(1);
+      const mongoHelper = await getMongoHelper(mongoUrl);
+      const databaseConnection = await mongoHelper.connect();
+      await mongoHelper.dropAllCollections();
+      await mongoHelper.given(nestedObjectModel);
+      const databaseAnalyzer = new DatabaseAnalyzer(databaseConnection, { dbDialect: 'mongodb' });
+      const model = await databaseAnalyzer.perform();
+      expect(model).toStrictEqual(expectedNestedObjectModel);
+      await mongoHelper.close();
```

Comment:
[IGNORABLE] This code looks close to others, we may have a function that encapsulate all. 
Something like `databaseAnalyzerTest(databaseConnection, fixtureModel, expectedModel)`